crates/swc_plugin_comments/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+authors = ["강동윤 <kdy1997.dev@gmail.com>", "OJ Kwon <kwon.ohjoong@gmail.com>"]
+description = "Internal sharable storage for the comments between host to the plugin"
+name = "swc_plugin_comments"
+version = "0.1.0"
+edition = "2021"
+publish = false
```

Comment:
Is this intentional?
If we do this, user can't add `swc_plugin` to their deps.

Reply:
No, probably mistake. Thanks for the catch, updated.